### ISSUE TITLE
Account lock system

### ISSUE THREAD

**Issue by Fyrsta** [TranxCraft/iTranxCraft#25]
A feature that will lock a players account by a password or something similar. This will need vigorous testing to make sure there isn't any security vulnerabilities with it.

**Comment by Wild1145** [comment]
I presume you are planning to use IP Verification to validate if the user has verified before?

**Comment by Fyrsta** [comment]
@Wild1145 If the player hasn't set up the account lock, it'll be able to be done by anybody. After they set it up and they do /lockaccount or something similar it'll ask for a password when the player logs in. IP verification won't be a thing here since it's not practical with those with dynamic IPs.

**Comment by Wild1145** [comment]
@WickedGamingUK Fair enough, I presume that just means you have to authenticate every time you join the server then?

**Comment by Fyrsta** [comment]
@Wild1145 Only if you enable it. You will be able to configure the system and if you want to lock your account then you can. Think of it like Windows, you can leave yourself logged on if you wish, but you can lock your account so you need a password. Somewhat a bad example, but you should be able to catch my drift.

**Comment by Kent55** [comment]
@WickedGamingUK Ah, like when they're logged in? This could be quite nice, if being implemented in the web panel too, we could add a nice JS lock screen.
